### PR TITLE
Use stats aggregated by client

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review and merge. Current implementation misses the call to `BenchmarkAggregator.start()`, so gives random time spent XD

### CODE REVIEW

File: vespa-feed-client/src/main/java/ai/vespa/feed/client/HttpRequest.java [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (bjorncs) [inline vespa-feed-client/src/main/java/ai/vespa/feed/client/HttpRequest.java:1]: ????

**Review by bjorncs — APPROVED** [review]

File: vespa-feed-client/src/main/java/ai/vespa/feed/client/HttpRequest.java (hunk 1 above)
Comment (jonmv) [reply vespa-feed-client/src/main/java/ai/vespa/feed/client/HttpRequest.java:1]: Hmm, what's wrong with this one? Thought it was the most recent? >_< IntelliJ uses that as a template for me, so I don't really notice when these appear.

Comment (bjorncs) [reply vespa-feed-client/src/main/java/ai/vespa/feed/client/HttpRequest.java:1]: I was wondering if this class was added intentionally or not

Comment (bjorncs) [reply vespa-feed-client/src/main/java/ai/vespa/feed/client/HttpRequest.java:1]: I probably looked at the code prior to force push.

**Pull request merged**